pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -32,25 +33,22 @@ let cmpFn = noSortFn;
 
 payload = d3.json(dataQueryUrl);
 payload.then(function (result) {
-    let svg = render(result);
+    let data = processAndSortResources(result);
+    let svg = render(data);
     bindMouseEvents(svg);
     appendAxes(svg);
     renderTooltip();
 });
 
-function render(result) {
-    let data = processAndSortResources(result);
+function render(data) {
```

Comment:
no objection - wondering about the **why** of pulling `processAndSortResources` out, I'm not seeing any other change that leverages this

Reply:
Mostly clarity and future use -esp for animations.